prompt2model/dataset_retriever/column_selection_prompt.py

```diff
@@ -0,0 +1,95 @@
+"""Utilities to construct an LLM "metaprompt" for our column selection."""
+
+from __future__ import annotations  # noqa FI58
+
+import json
+
+METAPROMPT_BASE = """Your objective is to carefully analyze the task, with the dataset mentioned and assign the columns into relevant input, output, ambigous and irrelevant columns for the given task. There should be at most one output column. It is possible to have no relevant columns, in which case return the input and output column as empty lists.  Answer in a json format, with the following keys: input, output, irrelevant, ambiguous"""  # noqa: E501
+METAPROMPT_EXAMPLES = [
+    (
+        """Your task is as follows. In this task, you will generate summaries for given texts. You will use the Scientific Papers dataset from HuggingFace. A sample data instance from this dataset is as follows.
```

Comment:
Were these examples actually created using the `build_input` function? If not, it may be a good idea to do so to, to make sure that the format of the in-context examples matches that of the inputs that the model will be getting.

Reply:
updated the examples!